### PR TITLE
Rerender without noarch

### PR DESCRIPTION
**Pull request by dgasmith**
@conda-forge-admin, please rerender

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,19 +13,19 @@ source:
...
 build:
   number: 2
-  script: "{{ PYTHON }} -m pip install . -vv"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
 
 requirements:
   host:
-  - python >=3.6
-  - pip
+    - python >=3.6
```
Comment (samuelcolvin) [inline recipe/meta.yaml]: sorry to be a pedant, while yaml is flexible about indents, technically lists don't need the extra indent you've added here.

Comment (dgasmith) [reply recipe/meta.yaml]: True, mostly shifting to align to general conda-forge style. Happy to move it back if you wish.

Comment (samuelcolvin) [reply recipe/meta.yaml]: ok, in that case, keep it like this.

**Comment by dgasmith** [comment]
@conda-forge/help-python, could use some help on this one.

```
Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
704WARNING:conda_build.build:Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
```

We can certainly install `conda-verify`, but I have yet to see a recipe where this is done. Any thoughts here?

**Comment by samuelcolvin** [comment]
fixes #3 (at least I'm guessing that's the idea here?)

**Comment by dgasmith** [comment]
@samuelcolvin Yes, that's is the idea here.

**Comment by Lnaden** [comment]
I think I found a fix for now.

* Unpin Python from the `host` and `run` sections
* Add `skip: True # [py2k]` to the `build` section
* Let the build system take care of it by only publishing Pydantic to the specific python versions

**Comment by dgasmith** [comment]
This is depending on conda-forge deprecating Py35?

**Comment by Lnaden** [comment]
You could also do `skip: True # [py2k or py35]` just to be sure?

**Comment by dgasmith** [comment]
@conda-forge-admin, please rerender

**Comment by Lnaden** [comment]
If the smithy did not re-render correctly, this may only build on one version... ugh.

**Comment by dgasmith** [comment]
Looks like this works for Py36/37 in testing. The commit might have stopped the render:

@conda-forge-admin, please rerender

**Pull request merged**